python/neuroglancer/pipeline/task_execution.py

```diff
@@ -1,14 +1,21 @@
+import sys
 from time import sleep
 import traceback
 
 from neuroglancer.pipeline import TaskQueue, EmptyVolumeException
 from neuroglancer.pipeline import logger
 
+# if an argument is supplied to this script
+# uses it as a tag to lease only certain kind of tasks
+tag = None
+if len(sys.argv) == 2:
```

Comment:
If someone calls the script with more than one argument, should it silently ignore all arguments (current behavior) or throw a warning / an error?